conf/node-default.properties

```diff
@@ -278,12 +278,15 @@
 # API.ServerEnforcePOST = yes
 
 ## Your keystore file and password, required if uiSSL or apiSSL are enabled.
+
 # API.SSL_keyStorePath     = keystore
 # API.SSL_keyStorePassword = password
 
 ## If you use https://certbot.eff.org/ to issue your certificate, provide below the path for your keys.
 ## BRS will automatically create the keystore file using the password above and will reload it weekly.
 ## Make sure you configure certbot to renew your certificate automatically so you don't need to worry about it.
+## Note, you need 'openssl' on your path for this to work, most Linux distributions have it already.
```

Comment:
dunno why this change, can be discarded, it was not intentional from me

Reply:
so remove it and update the commit